website/siteConfig.js

```diff
@@ -40,7 +40,7 @@ const siteConfig = {
 
   highlight: {
     // Highlight.js theme to use for syntax highlighting in code blocks
-    theme: 'vs2015',
+    theme: "github"
```

Comment:
I had picked vs2015 because the comments were very readable in the example. Maybe just make sure they still are, then 👍

Reply:
i picked github to be closer to the sites current look. I think they're readable but I guess that's a very subjective thing :) 

<img width="811" alt="screen shot 2018-07-25 at 21 30 52" src="https://user-images.githubusercontent.com/83561/43223029-13e8fb04-9052-11e8-8a65-faf61cb7f690.png">

Reply:
Looks readable enough! Although the last comment is not rendered properly for some reason

Reply:
Yeah that a bit funky - no idea why - probably it uses my textmate grammar and it's broken 😂I think we'll have to revisit all of these examples a bit when we port over the full configuration page so we can fix it then 😊